packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -75,8 +88,11 @@ export interface ISummarizeInternalResult extends ISummarizeResult {
 }
 
 /**
- * @experimental - Can be deleted/changed at any time
```

Comment:
A couple of notes: 

1. This interface is used (transitively) by a number of public items in this package. I tried marking this as `@alpha`, but that created a cascade of other items that would need to be marked as `@alpha`, which are a part of the public API.

2. The [@experimental](https://tsdoc.org/pages/tags/experimental/) tag is not a block tag - it does not support accompanying text content. Moved those contents to a `@remarks` block.

Reply:
I wouldn't block the PR on this but would definitely ask the author(s) of the interface what they think about this. Since it's de-facto public, I think making that explicit is correct but then we might need to analyze how it was used, to understand that pattern does not allow for truly @alpha/@experimental APIs.